libminifi/src/processors/PutFile.cpp

```diff
@@ -138,49 +138,18 @@ void PutFile::onTrigger(core::ProcessContext *context, core::ProcessSession *ses
     if (S_ISDIR(statResult.st_mode)) {
       // it's a directory, count the files
       int64_t ct = 0;
-#ifndef WIN32
-      DIR *myDir = opendir(directory.c_str());
-      if (!myDir) {
-        logger_->log_warn("Could not open %s", directory);
+      auto lambda = [&ct, this](const std::string&, const std::string&) -> bool {
```

Comment:
If we're going to use lambdas here we need a comment here about what the return means for the follow on function.

Reply:
Added.